### PR TITLE
Draft author list

### PR DESCRIPTION
**Pull request by adrn**
Here is my first attempt to produce the author list for the paper. In the first two author blocks, the authors are sorted by git "additions." The last block is sorted alphabetically by last name. I tried to group like affiliations by defining a new macro for these cases when it appears 2 or more times. Some people filled out the form twice, so I removed duplicates, but keep an eye out for other duplicates. I've already gone through and vetted that everyone who filled out the form has a commit or contribution to one of the astropy org repos.

~~Some people have a comment "%FLAG" above their name: these are people who submitted incomplete affiliations, so we will have to reach out to get full affiliations from them.~~
I updated all flagged affiliations by finding recent papers and grabbing affiliations, or just adding a bit more info to the affiliations (e.g., country).

### CODE REVIEW

File: src/static/authors.tex [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+\affiliation{Department of Astronomy, The University of Texas at Austin, 2515 Speedway, C1400, Austin, TX 78712, USA}
+
+\author[0000-0003-2657-3889]{Nicholas~Saunders}
+\affiliation{Institute for Astronomy, University of Hawaiʻi at M\=anoa, 2680 Woodlawn Drive, Honolulu, HI 96822, USA}
+\affiliation{American Museum of Natural History, 200 Central Park West, Manhattan, NY 10024, USA}
+\altaffiliation{NSF Graduate Research Fellow}
+
+%FLAG
+\author[0000-0001-6353-0808]{Volodymyr~Savchenko}
+\affiliation{University of Geneva, Department of Astronomy}
```
Comment (volodymyrss) [inline src/static/authors.tex:394]: I saw the PR, wanted to check if my affiliations are right, and realized it's flagged.

Is it ok to provide an update like this?

```suggestion
\affiliation{Department of Astronomy, University of Geneva, Chemin d'\'Ecogia, 16 CH-1290 Versoix, Switzerland}
\affiliation{EPFL SCITAS, Station 9, CH-1015 Lausanne, Switzerland}
```

Comment (adrn) [reply src/static/authors.tex:394]: Nice - thanks for being diligent 😄

File: src/static/authors.tex [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+
+\author[0000-0002-1838-4757]{Aniket~Sanghi}
+\affiliation{Department of Astronomy, The University of Texas at Austin, 2515 Speedway, C1400, Austin, TX 78712, USA}
+
+\author[0000-0003-2657-3889]{Nicholas~Saunders}
+\affiliation{Institute for Astronomy, University of Hawaiʻi at M\=anoa, 2680 Woodlawn Drive, Honolulu, HI 96822, USA}
+\affiliation{American Museum of Natural History, 200 Central Park West, Manhattan, NY 10024, USA}
+\altaffiliation{NSF Graduate Research Fellow}
+
+%FLAG
```
Comment (adrn) [inline src/static/authors.tex:392]:
```suggestion
```

**Comment by adrn** [comment]
Note: the build is failing because of a showyourwork bug - I think @rodluger and I will have to fix that upstream :)

**Pull request merged**